bugsnag-android-core/build.gradle

```diff
@@ -11,6 +11,10 @@ bugsnagBuildOptions {
 apply plugin: "com.android.library"
 apply plugin: "org.jetbrains.dokka"
 
+dependencies {
+    implementation 'com.dslplatform:dsl-json:1.9.8'
```

Comment:
The java8 stuff is only available in `dsl-json-java8` (if you use that by mistake it's still conditionally loaded).
It's BSD licensed.

Reply:
Example app size @5.4.0: 3145118
Example app size this branch: 3298843

Reply:
It's probably worth flagging out the size increase internally just so that everyone is aware, although it sounds like a reasonable difference - and we'll be able to remove `JsonWriter` which will grant some savings.

Reply:
Added PLAT-6950 for the unity note/